server/controllers/profile.js

```diff
@@ -0,0 +1,55 @@
+const router = require('express').Router();
+let Profile = require("../models/Profile");
+const { route } = require('../routes/auth');
+
+router.route('/').get((req, res) => {
+    Profile.find()
+      .then(profiles => res.json(profiles))
+      .catch(err => res.status(400).json('Error: ' + err));
+});
```

Comment:
Take a look at the other controller, see how they are setup. We define all our routes in the `routes` folder